### PR TITLE
refactor(scripts): Scripts should relay cross-chain governance to Arbitrum

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

I refactored a lot of the repeated logic into a `utils` file and I've been testing on local mainnet fork with admittedly simple test cases. Still need to test arbitrum in production but it's not possible right now since the arbitrum governance contracts are still owned by the dev wallet and the L2 governance contracts are also owned by this dev wallet.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [X]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

**Comment by nicholaspai** [comment]
@mrice32 @chrismaree the churn on this PR is very large but a lot of the files are getting cut a lot smaller. We probably shouldn't merge this in until the newly deployed contract owners are transferred over

**Comment by chrismaree** [comment]
> @mrice32 @chrismaree the churn on this PR is very large but a lot of the files are getting cut a lot smaller. We probably shouldn't merge this in until the newly deployed contract owners are transferred over

ok cool. will wait on reviewing as well.

File: packages/scripts/src/admin-proposals/addContractCreator.js [hunk 1]
```diff
@@ -52,146 +35,143 @@ const argv = require("minimist")(process.argv.slice(), {
...
-      await _getContractAddressByName("Registry", polygon_netId)
-    );
-    polygon_governor = new crossChainWeb3.eth.Contract(
-      GovernorChildTunnel.abi,
-      await _getContractAddressByName("GovernorChildTunnel", polygon_netId)
-    );
-  }
+  const networksToAdministrate = [];
+  if (polygon) networksToAdministrate.push(137);
+  if (arbitrum) networksToAdministrate.push(42161);
```
Comment (chrismaree) [inline packages/scripts/src/admin-proposals/addContractCreator.js:44]: so we'll need to add optimism and boba here as well?

File: packages/scripts/src/admin-proposals/addContractCreator.js [hunk 2]
```diff
@@ -52,146 +35,143 @@ const argv = require("minimist")(process.argv.slice(), {
...
+        });
       } else {
-        console.log("- Contract @ ", polygonContractToRegister, "is already a contract creator. Nothing to do.");
+        console.log("- Contract @ ", polygon, "is already a contract creator. Nothing to do.");
+      }
+
+      console.groupEnd();
+    }
+
+    if (arbitrum) {
```
Comment (chrismaree) [inline packages/scripts/src/admin-proposals/addContractCreator.js:134]: kinda sucks that we need all this chain specific case work 🤔

**Review by chrismaree — APPROVED** [review]
this is a lot. I tried to go over all the changes but there really is a bug diff here. overall from what I examined this all seems great! I wish there was a way to have less chain specific case work though. any ideas on how to generalise this more?

File: packages/scripts/src/admin-proposals/addContractCreator.js (hunk 1 above)
Comment (nicholaspai) [reply packages/scripts/src/admin-proposals/addContractCreator.js:44]: yep

**Comment by nicholaspai** [comment]
> this is a lot. I tried to go over all the changes but there really is a bug diff here. overall from what I examined this all seems great! I wish there was a way to have less chain specific case work though. any ideas on how to generalise this more?

I'm working on some refactors will re-request when done

**Pull request merged**